src/app/users/utils.js

```diff
@@ -291,7 +299,7 @@ const getAllServicesForUserInOrg = async (
   const services = userServicesForOrg.map((service) => ({
     id: service.serviceId,
     dateActivated: service.accessGrantedOn,
-    name: '',
+    name: "",
```

Comment:
Should be single quotes to be consistent with existing source code.

Reply:
Thanks Lea, I did install prettier at one point and i think it's been playing around with my linter.